haskell/private/actions/package.bzl

```diff
@@ -96,9 +96,7 @@ def package(hs, dep_info, interfaces_dir, interfaces_dir_prof, static_library, d
         command = """
         cat $1 > $4
         echo "exposed-modules: $(< $2)" >> $4
-        echo -n "depends: " >> $4
-        cat $3 | tr "\\n" " " >> $4
-        echo >> $4
+        echo "depends: $(cat $3 | tr "\\n" " ")" >> $4
```

Comment:
Squash into last change?

Reply:
(This is on the very last commit, Github’s review feature is very bad with multi-commit PRs)

Reply:
> (This is on the very last commit, Github’s review feature is very bad with multi-commit PRs)

It is.

> Squash into last change?

Done